help/sources/tutorials/ui/create/analytics/pendo-webhook.md

```diff
@@ -0,0 +1,165 @@
+
+Platform provides intelligent recommendations for auto-mapped fields based on the target schema or dataset that you selected. You can manually adjust mapping rules to suit your use cases. Based on your needs, you can choose to map fields directly, or use data prep functions to transform source data to derive computed or calculated values. For comprehensive steps on using the mapper interface and calculated fields, see the [Data Prep UI guide](https:/experienceleague.adobe.com/docs/experience-platform/data-prep/ui/mapping.html).
+
+The mappings listed below are mandatory and should be setup before proceeding to the [!UICONTROL Review] stage.
+
+| Target Field | Description |
+| --- | --- |
+| `uniqueId` | [!DNL Pendo] identifier for the event. |
+
+Once your source data is successfully mapped, select **[!UICONTROL Next]**.
+
+![The mapping step of the sources workflow.](../../../../images/tutorials/create/analytics-pendo-webhook/mapping.png)
+
+## Review
+
+The **[!UICONTROL Review]** step appears, allowing you to review your new dataflow before it is created. Details are grouped within the following categories:
+
+* **[!UICONTROL Connection]**: Shows the source type, the relevant path of the chosen source file, and the amount of columns within that source file.
+* **[!UICONTROL Assign dataset & map fields]**: Shows which dataset the source data is being ingested into, including the schema that the dataset adheres to.
+
+Once you have reviewed your dataflow, click **[!UICONTROL Finish]** and allow some time for the dataflow to be created.
+
+![The review step of the sources workflow.](../../../../images/tutorials/create/analytics-pendo-webhook/review.png)
+
+## Get your streaming endpoint URL {#get-streaming-endpoint-url}
+
+With your streaming dataflow created, you can now retrieve your streaming endpoint URL. This endpoint will be used to subscribe to your webhook, allowing your streaming source to communicate with Experience Platform. 
+
+Next, you will require to construct a URL using the API details generated within Platform which will be used to configure the webhook on [!DNL Pendo].
+
+Note down the values of the fields below: 
+* **[!UICONTROL Dataflow ID]**
+* **[!UICONTROL Streaming endpoint]**
+
+To retrieve your **[!UICONTROL Dataflow ID]** and **[!UICONTROL Streaming endpoint]**, go to the [!UICONTROL Dataflow activity] page of the dataflow that you just created and copy the details from the bottom of the [!UICONTROL Properties] panel.
+
+![The streaming endpoint in dataflow activity.](../../../../images/tutorials/create/analytics-pendo-webhook/endpoint-test.png)
+
+Construct a URL using these values following the pattern below:
+```{STREAMING_ENDPOINT}?x-adobe-flow-id={DATAFLOW_ID}```
+
+Example of a constructed webhook URL: ```https://dcs.adobedc.net/collection/0c61859cc71939a0caf01123f91b2fc52589018800ad46b6c76c2dff3595ee95```
```

Comment:
Let's revise to:
Once you have retrieved your streaming endpoint and dataflow ID, build a URL based on the following pattern: `{STREAMING_ENDPOINT}?x-adobe-flow-id={DATAFLOW_ID}`. For example, a constructed webhook URL may look like: `https://dcs.adobedc.net/collection/0c61859cc71939a0caf01123f91b2fc52589018800ad46b6c76c2dff3595ee95`